lib/jormungandr/test/data/jormungandr/stake_pools/create-stake-pool.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+# Usage:
+# Run in the stake_pools directory, where owner.prv exists.
+#
+# To create a stake pool in a folder "a":
+# .create-stake-pool.sh a
+
+echo "creating certificates for a new stake pool in the folder $1"
+mkdir $1
+cd $1
+
+jcli key generate --type=Curve25519_2HashDH > stake_pool_vrf.prv
+cat stake_pool_vrf.prv | jcli key to-public > stake_pool_vrf.pub
```

Comment:
The more normal way is to use `< stake_pool_crf.prv` instead of `cat`.

Reply:
Couldn't get it working. Besides, `jormungandr` documentation uses `cat file | jcli ...` syntax extensively.

Reply:
This is what I meant: [Combining redirections](https://www.tldp.org/LDP/intro-linux/html/sect_05_01.html#sect_05_01_02_03).

Either one works and is fine but it's shorter to not use `cat` when it's not necessary.

Reply:
Ok, figured it out. Cool. E.g `jcli key to-public < account1.prv`
 
I'll stick to `cat` for consistency with the jormungandr docs and since I'm used to it, and also, I think the way `cat x | f | g | h` chains is a bit neater than `f < x | g | h`